pkg/scorecard.go

```diff
@@ -90,6 +96,21 @@ func RunScorecard(ctx context.Context,
 	ossFuzzRepoClient clients.RepoClient,
 	ciiClient clients.CIIBestPracticesClient,
 	vulnsClient clients.VulnerabilitiesClient,
+) (ScorecardResult, error) {
+	return runScorecardChecksV5(ctx, repo, commitSHA, commitDepth,
+		checksToRun, repoClient, ossFuzzRepoClient,
+		ciiClient, vulnsClient)
+}
+
+func runScorecardChecksV5(ctx context.Context,
```

Comment:
this still seems like an unneeded change to me, but not going to block on it.

Reply:
removed.